test/framework/testsuite.go

```diff
@@ -200,7 +200,12 @@ func (config *TestsuiteConfig) RunAllTests(logdir string) int {
 	LogInfo("TESTS EXECUTED:", totalTests)
 	LogInfo("PASSED:", passedTests)
 	LogInfo("FAILED:", failedTests)
-	return failedTests
+
+	if failedTests > 0 {
+		return 100 + failedTests
```

Comment:
Why 100 + tests? Is this for CI/CD?

Reply:
Yes, PR comment explains it. It is to easily distinguish common build/run errors with codes 1-10 from number of failed tests just by looking at the error code.